DevCycle.SDK.Server.Local/ConfigManager/EnvironmentConfigManager.cs

```diff
@@ -150,8 +157,26 @@ private async Task FetchConfigAsyncWithTask()
             {
                 if (Config == null && configEtag == null)
                 {
-                    logger.LogError("Error loading initial config. Exception: {Exception}", e.Message);
                     dvcEventArgs.Error = e;
+                }
+                
+                if (!e.IsRetryable())
+                {
+                    if ((int)e.HttpStatusCode == 403)
+                    {
+                        logger.LogError("Project configuration could not be found. Check your SDK key.");
```

Comment:
should these new logs be added as errors to dvcEventArgs so the sdk user will see it via the initialized notification event?

Reply:
the logic here before was to only attach the error to EventArgs if the config hadn't been set yet.

Given your new OnInitialize logic, can I just attach the error to EventArgs here no matter what and assume that it's not going to send it back to the user unless the config hasn't been retrieved yet?

Reply:
EventArgs probably should be expanded to include eventtypes so that we can notify of a mid-run config failure to fetch because of whatever reasoning, as well as a variable defaulting reasons.

Reply:
i think that's outside the scope of this, but I've updated this block to send back the DVCException with the correct message in all cases